mockspec/rules

```diff
@@ -7,6 +7,16 @@ k8s; pkg/k8s/mock_controller_generated.go; github.com/openservicemesh/osm/pkg/k8
 
 # pkg/debugger
 debugger; pkg/debugger/mock_debugger_generated.go; github.com/openservicemesh/osm/pkg/debugger; XDSDebugger
+debugger; pkg/debugger/mock_debugger_infra_client_generated.go; github.com/openservicemesh/osm/pkg/debugger; DebuggerInfraClient
 
-# pkg/compute
-compute; pkg/compute/mock_compute_client_generated.go; github.com/openservicemesh/osm/pkg/compute; Interface
+# pkg/catalog
+catalog; pkg/catalog/mock_compute_interface_generated.go; github.com/openservicemesh/osm/pkg/catalog; Interface
+
+# pkg/injector
+injector; pkg/injector/mock_injector_infra_client_interface_generated.go; github.com/openservicemesh/osm/pkg/injector; InjectorInfraClient
```

Comment:
injector.InjectorInfraClient stutters, and it's not used outside of the package. So you can unexport it, and just call it infraClient.

Same for the otheres